common/web/keyboard-processor/src/text/codes.ts

```diff
@@ -1,7 +1,7 @@
 // TODO:  Move to separate folder:  'codes'
 // We should start splitting off code needed by keyboards even without a KeyboardProcessor active.
-// There's an upcoming `/common/web/types` package that 'codes' and 'keyboards' may fit well within.
-// In fact, there's a file there (on its branch) that should be merged with this one!
```

Comment:
There is a TODO for this. we haven't merged these yet.

Reply:
right. was just trying to update it